concent_api/core/migrations/0006_add_and_update_deadline_and_size.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-06-19 09:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+from common.helpers import deserialize_message
+from common.helpers import parse_timestamp_to_utc_datetime
+
+
+def populate_subtask_size_field(apps, _schema_editor):
+    Subtask = apps.get_model('core', 'Subtask')
+    for subtask in Subtask.objects.filter(result_package_size__isnull=True):
+        subtask.result_package_size = deserialize_message(subtask.report_computed_task.data).size
+        subtask.full_clean()
+        subtask.save()
+
+
+def populate_subtask_deadline_field(apps, _schema_editor):
+    Subtask = apps.get_model('core', 'Subtask')
+    for subtask in Subtask.objects.filter(computation_deadline__isnull=True):
+        subtask.computation_deadline = parse_timestamp_to_utc_datetime(
+            deserialize_message(
+                subtask.task_to_compute.data
+            ).compute_task_def['deadline']
+        )
+        subtask.full_clean()
+        subtask.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('core', '0005_auto_20180717_0908'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='subtask',
+            name='computation_deadline',
+            field=models.DateTimeField(blank=True, null=True),
```

Comment:
Same as in https://github.com/golemfactory/concent/pull/570#discussion_r203621222, there is mismatch between field definition and migration. What is the reason for it ?

Reply:
It was wrong. Already changed and divided to 2 migrations.